src/CMakeLists.txt

```diff
@@ -83,6 +83,9 @@ ENDIF()
 
 add_executable(NimbleSM_Serial nimble_serial_main.cc)
 target_link_libraries(NimbleSM_Serial PRIVATE nimble::nimble)
+if (NIMBLE_HAVE_KOKKOS)
+  target_link_libraries(NimbleSM_Serial PRIVATE Kokkos::kokkos)
```

Comment:
This does not build for me with Kokkos from Trilinos. I need to do something similar to what you've got below for handling NimbleSM_Kokkos, i.e., if I have Trilinos, I need to link Trilinos::Trilinos, because Kokkos::kokkos is not a known target.

Reply:
Done.